tests/integration/test_fim/test_synchronization/test_sync_overlap.py

```diff
@@ -0,0 +1,193 @@
+        2: Only level 2 tests are performed, they check advanced functionalities and are slow to perform.
+
+tags:
+    - fim_synchronization
+'''
+import os
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.tools import LOG_FILE_PATH, configuration
+from wazuh_testing.tools.monitoring import FileMonitor, generate_monitoring_callback
+from wazuh_testing.modules import TIER2, AGENT, SERVER
+from wazuh_testing.modules import fim
+from wazuh_testing.fim import callback_detect_synchronization
+
+# Marks
+pytestmark = [AGENT, SERVER, TIER2]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+test_cases_path = os.path.join(TEST_CASES_PATH, 'cases_sync_overlap.yaml')
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_sync_overlap.yaml')
+
+# Test configurations
+configuration_parameters, configuration_metadata, test_case_ids = configuration.get_test_cases_data(test_cases_path)
+for count, value in enumerate(configuration_parameters):
+    configuration_parameters[count]['MONITORED_DIR'] = fim.MONITORED_DIR_1
+configurations = configuration.load_configuration_template(configurations_path, configuration_parameters,
+                                                           configuration_metadata)
+
+# Variables
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Tests
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=test_case_ids)
+@pytest.mark.parametrize('files_number', [configuration_metadata[0]['files']])
+def test_sync_overlap(configuration, metadata, set_wazuh_configuration_fim, create_files_in_folder,
+                      restart_syscheck_function, wait_for_fim_start_function):
+    '''
+    description: Check if the 'wazuh-syscheckd' daemon is performing a synchronization at the interval specified in the
+                 configuration, using the 'interval' tag, if a new synchronization is fired, and the last sync message
+                 has been recieved in less time than 'response_timeout, the sync interval is doubled.
+                 The new value for interval cannot be higher than max_interval option. After a new sync interval is
+                 tried and the last message was recieved in a time that is higher than response_timeout, the sync
+                 interval value is returned to the configured value.
+
+    test_phases:
+        - Create a folder with a number of files inside.
+        - Restart syscheckd.
+        - Check that a sync interval started.
+        - Check that next sync is skipped and interval value is doubled
+        - Check that interval value is returned to configured value after successful sync
+
+    wazuh_min_version: 4.5.0
+
+    tier: 2
```

Comment:
This test should be `tier 1`.

Reply:
Fixed

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3380/commits/f21ac15dc4c1aceb816f099b449e4749932183af